.github/workflows/ci.yml

```diff
@@ -106,7 +106,40 @@ jobs:
               libdbus-1-dev",
             build_type: "Release"
           }
-
+        - {
+            name: "Android_armeabi-v7a-Release",
+            os: ubuntu-latest,
+            cmake_opts: "-DCMAKE_TOOLCHAIN_FILE=/usr/local/lib/android/sdk/ndk/26.3.11579264/build/cmake/android.toolchain.cmake \
```

Comment:
Out of curiosity, where does the SDK and version number come from? It doesn't look like anything's installing it, and that version looks like it's prone to change. And are any of the apt packages needed for the Android build?

Reply:
GitHub actions runners come with NDK pre-installed, but its directory variable wasn't defined so I had to hardcode the path after finding it by recursively listing all files in the drive. Kinda hacky, I know lol
But I'll see if I can find a way to set/get the proper variable for a cleaner/relative path. 👌 

As for the packages, I'm not sure if they're all needed. I just used the linux build settings as a template since that's technically what Android is based on and what the runner uses, but I'll try building without them, though I'm not sure it'd inadvertently break something so let me know if there's any packages that are probably needed for core functions.

Reply:
I don't think any of the packages should be needed. I don't think they'd work anyway since they're for the system it's compiling on, which wouldn't work for the Android target (though it looks like it may be picking up the pipewire and dbus pkg-configs, which it shouldn't because Android doesn't have them as far as I know; luckily it doesn't cause a build failure since they're optional at runtime).

Reply:
I just looked at the [log](https://github.com/user-attachments/files/16482376/log_2024_08_03_13_35_52.log) and both indeed fail to load/initialize so I'll go ahead and remove the whole `deps_cmdline` chunk and see how it goes. 🤞

Reply:
btw just found the proper NDK path variable https://github.com/kcat/openal-soft/pull/1025/commits/6538bff88dcd7b0ecf6c47c8e2c46ce1d95d5721 👌